business/services.go

```diff
@@ -230,3 +230,48 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 	s.SetSourceWorkloads(sWk)
 	return &s, nil
 }
+
+// GetServiceDefinition returns a single service definition (the service object and endpoints), no istio or runtime information
+func (in *SvcService) GetServiceDefinition(namespace, service string) (*models.ServiceDetails, error) {
```

Comment:
Instead to repeat the code in GetServiceDefinition I suggest to move the common one into a helper function and use the helper both in GetService / GetServiceDefinition.